xcode/Subconscious/Shared/AppIcon.swift

```diff
@@ -8,7 +8,7 @@
 import SwiftUI
 
 enum AppIcon {
-    case you
+    case you(ColorScheme)
```

Comment:
I started going down a rabbithole of picking new icons and realised it's not **that** much work to pick the right smiley. We're not the only ones to hit this issue and fix it this way: https://github.com/Dimillian/IceCubesApp/issues/1193